internal/pkg/docker/dockercompose/decompose_test.go

```diff
@@ -143,6 +143,12 @@ func TestDecomposeService(t *testing.T) {
 				},
 			},
 		},
+		"multiple env files": {
```

Comment:
🤔 Since we're already breaking down the env files into vars, couldn't we loop through multiple env files and combine their contents (filtering out duplicates)? We'd have to handle clashes/overrides.